superset/tasks/cache.py

```diff
@@ -75,13 +73,13 @@ def get_form_data(chart_id, dashboard=None):
     return form_data
 
 
-def get_url(params):
+def get_url(chart):
     """Return external URL for warming up a given chart/table cache."""
-    baseurl = "http://{SUPERSET_WEBSERVER_ADDRESS}:{SUPERSET_WEBSERVER_PORT}/".format(
```

Comment:
`http://` here seemed bad to me as someone might have `https://` decided to remove it @betodealmeida